<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionFrame</class>
 <widget class="QFrame" name="PrescriptionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>613</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="medicationBox">
     <property name="title">
      <string notr="true">Prescriptions search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="medicationNameLabel">
          <property name="toolTip">
           <string notr="true">Name of the medication</string>
          </property>
          <property name="text">
           <string notr="true">Medication</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="medicationNameField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string notr="true">Name of the medication</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lotLabel">
          <property name="toolTip">
           <string notr="true">Lot number</string>
          </property>
          <property name="text">
           <string notr="true">Lot #</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lotField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string notr="true">Lot number</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="filledLabel">
          <property name="toolTip">
           <string notr="true">Filled</string>
          </property>
          <property name="text">
           <string notr="true">Filled</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="filledField">
          <property name="toolTip">
           <string notr="true">Filled</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1970</year>
            <month>1</month>
            <day>31</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="invalidField">
          <property name="text">
           <string notr="true">Invalid</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="patientBox">
     <property name="title">
      <string notr="true">Patient search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lastLabel">
          <property name="toolTip">
           <string notr="true">Last name</string>
          </property>
          <property name="text">
           <string notr="true">Last name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string notr="true">Last name</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="firstLabel">
          <property name="toolTip">
           <string notr="true">First name</string>
          </property>
          <property name="text">
           <string notr="true">First name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string notr="true">First name</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="dobLabel">
          <property name="toolTip">
           <string notr="true">Date of birth</string>
          </property>
          <property name="text">
           <string notr="true">DOB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dobField">
          <property name="toolTip">
           <string notr="true">Date of birth</string>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1970</year>
            <month>1</month>
            <day>31</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="activeField">
          <property name="text">
           <string notr="true">Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string notr="true">Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string notr="true">Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="DisplayTable" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>14</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true"/>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">prescriptions.id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">MR Number</string>
      </property>
      <property name="toolTip">
       <string notr="true">MR Number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Last name</string>
      </property>
      <property name="toolTip">
       <string notr="true">Last name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">First name</string>
      </property>
      <property name="toolTip">
       <string notr="true">First name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">DOB</string>
      </property>
      <property name="toolTip">
       <string notr="true">Date of birth</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Medication</string>
      </property>
      <property name="toolTip">
       <string notr="true">Name of the medication</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Form</string>
      </property>
      <property name="toolTip">
       <string notr="true">Medication form</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Strength</string>
      </property>
      <property name="toolTip">
       <string notr="true">Strength</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Amount</string>
      </property>
      <property name="toolTip">
       <string notr="true">Amount</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Written</string>
      </property>
      <property name="toolTip">
       <string notr="true">Written</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Filled</string>
      </property>
      <property name="toolTip">
       <string notr="true">Filled</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Lot #</string>
      </property>
      <property name="toolTip">
       <string notr="true">Lot number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Status</string>
      </property>
      <property name="toolTip">
       <string notr="true">Valid or invalid status</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="printAction">
   <property name="text">
    <string notr="true">Print Label</string>
   </property>
   <property name="toolTip">
    <string notr="true">Print a label for the prescription</string>
   </property>
  </action>
  <action name="invalidateAction">
   <property name="text">
    <string notr="true">Invalidate</string>
   </property>
   <property name="toolTip">
    <string notr="true">Marks the prescription as invalid.</string>
   </property>
  </action>
  <action name="viewAction">
   <property name="text">
    <string notr="true">View Prescription</string>
   </property>
   <property name="toolTip">
    <string notr="true">View prescription record information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayTable</class>
   <extends>QTableWidget</extends>
   <header>displaytable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>medicationNameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lotField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lastField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>firstField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>medicationNameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lotField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
